skyrim-platform/src/platform_se/skyrim_platform/TextApi.h

```diff
@@ -22,5 +29,12 @@ JsValue GetTextString(const JsFunctionArguments& args);
 
 JsValue GetTextColor(const JsFunctionArguments& args);
 
+JsValue GetTextSize(const JsFunctionArguments& args);
+JsValue GetTextRotation(const JsFunctionArguments& args);
+JsValue GetTextFont(const JsFunctionArguments& args);
+JsValue GetTextDepth(const JsFunctionArguments& args);
+JsValue GetTextEffect(const JsFunctionArguments& args);
+JsValue GetTextOrigin(const JsFunctionArguments& args);
```

Comment:
Looks like there are blank lines between funcs everywhere else in this file

Reply:
fixed